src/shared/processors/low/wpappbox.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
```

Comment:
Did these components need any change? If positive, had all of them been tested?

If these components are working like before and just moved the code to the new folder, don't mind this comment.